### PR TITLE
Make draft-email-campaign-frontend available

### PR DESCRIPTION
**Pull request by deanwilson**
This vhost wasn't set, which means nothing is listening to this vhost.

@bradwright @mattbostock and I looked at the previous PR and this seems to fix the tests. Opened as a new PR in case it's wrong.

### CODE REVIEW

File: hieradata/development.yaml [hunk 1]
```diff
@@ -101,6 +101,7 @@ govuk::node::s_development::apps:
 govuk_app_enable_capistrano_layout: false
 govuk_app_enable_services: false
 
+govuk::apps::email_campaign_frontend::vhost: 'draft-email-campaign-frontend'
```
Comment (mattbostock) [inline hieradata/development.yaml]: This is specific to draft, which might be confusing (I think non-draft would be better as a default).

Comment (bradwright) [inline hieradata/development.yaml]: Updated and force pushed.

**Comment by bradwright** [comment]
I know this pattern just follows the others, but I was wondering why we don't set the default `vhost` (eg `email-campaign-frontend` or `government-frontend`) in the Puppet class as a default argument value? Then we'd only need to override it on draft machines and not explicitly set it everywhere.

**Comment by alexmuller** [comment]
I can't think of a good reason to not set a default. I think that would be the correct thing to do in this case.

**Comment by bradwright** [comment]
Okay, so I'll do the default vhost refactor, but as a separate PR as this one seems functionally complete.

**Pull request merged**